ingestion/src/metadata/ingestion/source/database/bigquery/metadata.py

```diff
@@ -430,6 +430,12 @@ def close(self):
         if self.temp_credentials:
             os.unlink(self.temp_credentials)
         os.environ.pop("GOOGLE_CLOUD_PROJECT", "")
+        if isinstance(
+            self.service_connection.credentials.gcpConfig, GcpCredentialsValues
+        ) and ("GOOGLE_APPLICATION_CREDENTIALS" in os.environ):
```

Comment:
let's just reuse the constant `GOOGLE_CREDENTIALS` that we have defined in utils.credentials